tests/Support/SupportCollectionTest.php

```diff
@@ -558,6 +558,28 @@ public function testMap()
         $this->assertEquals(['first' => 'first-rolyat', 'last' => 'last-llewto'], $data->all());
     }
 
+    public function testWalkWithModification()
+    {
+        $data = new Collection([['color' => 'red'],['color' => 'blue']]);
```

Comment:
Interesting that StyleCI didn't complain about that. I need to get a PSR2 plugin in ST3 so don't have to make such simple mistakes.